charts/kubewarden-defaults/tests/failure_policy_test.yaml

```diff
@@ -0,0 +1,25 @@
+suite: set failurePolicy based upon mode
+templates:
```

Comment:
I'm thinking if we should also add a more generic test.
If we don't want any policies to have policy Fail right now, should we add a test to get all created policies and check? but where would that that run? best way I can think is a new module for smoke testing the platform after all modules deploy.
Do we then want still to have Ignore when the policies aren't in monitor mode anymore? should we fail in favor of the client?

Reply:
I think I'll just approve this PR but we need to have a think on how to make future policies compliant. Creating a policy to validate policies already seems silly lol